### PR TITLE
Making my specs better

### PR DESCRIPTION
**Pull request by akampjes**
Part 1 ;)

### CODE REVIEW

File: spec/services/fulfill_order_spec.rb [hunk 1]
```diff
@@ -42,9 +42,21 @@
     expect(sell_order).to be_fulfilled
   end
 
-  # Test that money is transfered (the obvious side-effect of TransferMoneyForFill
-  it 'transfers some money after fulfilling an order' do
-    expect { subject.call }.to change {buy_order.user.money}
-    expect { subject.call }.to change {sell_order.user.money}
+  context 'money is transfered' do
```
Comment (Widdershin) [inline spec/services/fulfill_order_spec.rb]: transfered -> transferred

**Comment by akampjes** [comment]
@Widdershin okay, my specs are now amazing and ready for review...
No rush, got other things todo :)

File: spec/services/create_order_spec.rb [hunk 2]
```diff
@@ -9,20 +9,22 @@
...
+      subject.call
+
+      expect(MatchOrdersJob).to have_been_enqueued.once.with(deserialize_as(stock: stock))
     end
   end
 
-  context 'invaild orders' do
-    it 'adds errors if a user doesnt have enough money to buy shares' do
+  context 'with an invaild order' do
+    it 'is invaild without enough money' do
```
Comment (Widdershin) [inline spec/services/create_order_spec.rb]: invaild -> invalid

**Comment by Widdershin** [comment]
Nice, I like your usage of describe/context/it a lot more nowadays, feels much more idiomatic.

We should write a poem with rspec that describes and gives context for idiomatic usage :smile: 

Merge!

**Pull request merged**